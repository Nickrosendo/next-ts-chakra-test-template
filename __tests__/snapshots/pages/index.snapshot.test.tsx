import React from "react";
import renderer from "react-test-renderer";
import Index from "@root/pages/index";

it("should render homepage matching snapshot", () => {
  const tree = renderer.create(<Index />).toJSON();
  expect(tree).toMatchSnapshot();
});
