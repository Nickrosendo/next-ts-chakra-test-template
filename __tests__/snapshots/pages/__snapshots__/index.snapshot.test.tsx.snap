// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render homepage matching snapshot 1`] = `
<div
  className="css-h74a88"
  role="main"
>
   
  <h2
    className="chakra-heading css-xi606m"
  >
     hello world 
  </h2>
</div>
`;
